### PR TITLE
Poison knife

### PR DESCRIPTION
**Pull request by enthethan**
## About The Pull Request
ports Seris02's  "Poison knife" PR 
https://github.com/Citadel-Station-13/Citadel-Station-13/pull/10797

## Why It's Good For The Game
new traitor item 

## Changelog
:cl:
add: Poison knife, a knife available to syndies for 5 tc which injects 3-5 chems and does 12 brute damage.
imageadd: sprite for the knife
/:cl:

### CODE REVIEW

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -55,6 +56,30 @@
...
+	create_reagents(40,OPENCONTAINER)
+
+/obj/item/kitchen/knife/poison/attack(mob/living/M, mob/user)
+	if (!istype(M))
+		return
+	. = ..()
+	if (!reagents.total_volume || !M.reagents)
+		return
+	var/randvar = pick(3,5)
+	var/amount = min(randvar/reagents.total_volume,1)
```
Comment (qwertyquerty) [inline code/game/objects/items/kitchen.dm:80]: Can you explain your thought process for the math here?

Comment (enthethan) [reply code/game/objects/items/kitchen.dm:80]: >ports the poison knife from citadel

🤷‍♂

Comment (enthethan) [reply code/game/objects/items/kitchen.dm]: although i see that line 80 is redundant

Comment (qwertyquerty) [reply code/game/objects/items/kitchen.dm:80]: Well what is it meant to do

Comment (enthethan) [reply code/game/objects/items/kitchen.dm:80]: well, var/randvar picks 3 4 or 5 units.(90% sure that (3,5) is 3-5 not 3 or 5))
var/amount is the inject amount which is decided by how much var/randvar picks. i guess
still man reeee merge it i tested it, it works no runtimes no poo shit cum its great now merje.

Comment (qwertyquerty) [reply code/game/objects/items/kitchen.dm:80]: But why does it divide by reagents.total_volume

**Comment by qwertyquerty** [comment]
Appears abandoned

**Comment by enthethan** [comment]
code works, it's tested and it works as intended , passes checks, and vote succedeed why is this closed?

**Pull request closed without merging**